### DISCUSSION TITLE
Any Idea for Dynamic Extending in Volt Templates?

### DISCUSSION THREAD

**Discussion by AliShareei** [phalcon/cphalcon#15284] [Q&A] [answered]
I searched for hours on this but got no results. This question has been asked many times before but there seems to be no solution!

See this: https://forum.phalcon.io/discussion/3185/extending-using-a-dynamic-value

In fact, I want to specify the layouts according to the settings in the app. For example:
```
{% extends "/layouts/"~ {{ layout_name_variable }} ~".volt" %}
{% block content %}
	{{ content() }}
{% endblock %}
```
Is there a specific way to implement this feature? Any suggestions can be helpful.

**Comment by Jeckerson** [comment, marked as answer]
Hello

It will be implemented in future versions.
Progress can be tracked here - https://github.com/phalcon/cphalcon/issues/14411

ETA is unknown.

**Reply by AliShareei** [reply]
Hi @Jeckerson 
I was a little bit disappointed for this issue but this is great news! Thanks.
